blimflix/client/app/core/sidebar.scss

```diff
@@ -0,0 +1,75 @@
+.sidebar-wrapper {
+height: 100%;
+background-color: #222;
+position: fixed;
+left: 0;
+top: 50px;
+padding: 0;
+padding-top: 70px;
+border-right: 2px solid black;
+}
+
+.profile-wrapper {
+height: 50px;
+color: #FFFFFF;
+}
+
+.row {
+    color: #FFFFFF;
+}
+
+.row img {
+    width: 100%;
+    height: 100%;
+}
+
+.row h4 {
+    font-weight: bold;
+}
+
+.row h4, .row h5 {
+    padding: 0;
+    margin:  0;
+    padding-top: 5px;
+    padding-left: 10px
+}
+
+.col-centered{
```

Comment:
Same thing as Manu says.
you will use the bootstrap grid classes in another places, 
It's better that you will override only style classes (btn, btn-danger...)
but avoid override estructural classes (.row, .col-md-3, .container...)